theano/tensor/basic.py

```diff
@@ -6093,3 +6093,12 @@ def c_code_cache_version(self):
 
     def do_constant_folding(self, node):
         return False
+
+    def connection_pattern(self, node):
+        return [[False] for i in node.inputs]
+
+    def grad(self, inputs, grads):
+        return [DisconnectedType()() for i in inputs]
+
+    def R_op(self, inputs, eval_points):
+        return [zeros(inputs, self.dtype)]
```

Comment:
@lamblin : I know the gradients have restrictions on their dtypes (they can't be int, if I recall correctly). Do the outputs of the R_op function have the same restrictions? If yes, I will need to change this implementation which would be incorrect if AllocEmpty was used with an integer dtype.